fastembed/models.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "name": "BAAI/bge-small-en-v1.5",
+        "sources": [
+            "Qdrant/bge-small-en-v1.5-onnx-Q"
+        ]
+    },
+    {
+        "name": "BAAI/bge-base-en-v1.5",
+        "sources": [
```

Comment:
The GCS sources should be mentioned here instead of constructing the URL from the model name/id.

Reply:
I've kept the download the GCS download method's signature and implementation as is, since we'll be dropping it, didn't want to do any tinkering and unexpectedly break something.

Reply:
We want to keep the GCS support — please see the discussion in the issue. We don't want to drop it right now.

Reply:
We want to retain URL support — allows folks to use internal URLs with fastembed! 

So two things actually: 

1. Rename our GCS utils to focus on the URL part
2. Write it with the view that we'll support the URL mode for a lot more time!

Does that make it clear that we're not going to drop GCS right now/soon? Eventually might be several releases away

Reply:
How can we ensure an arbitrary URL consists the appropriate model files with the appropriate names?
The dir may or may not be zipped or structured as expected.

Reply:
We'll have to test for that. We don't test right now because we were the only ones using that. If we want to allow arbitrary URL mode, the lib itself will have to check these cases e.g.

1. Does it have the valid model files? 
2. Is the format correct e.g. ONNX instead of ONNX.data if that's an issue? 
3. Is the compressed file okay? E.g. format tar.gz instead of something else? 

And so on